bucket/impregnate.json

```diff
@@ -0,0 +1,27 @@
+{
+    "version": "1.2.2",
+    "description": "A standalone installer for Cumcord.",
```

Comment:
I would also recommend expanding on the definition and making it clear what the tool this installs does.